vm/array.go

```diff
@@ -735,6 +735,22 @@ func builtinArrayInstanceMethods() []*BuiltinMethodObject {
 				}
 			},
 		},
+		{
+			// Inserts the specified element in the first position of the array.
+			//
+			// ```ruby
+			// a = [1, 2]
+			// a.unshift(0) # => [0, 1, 2]
+			// a       # => [0, 1, 2]
```

Comment:
Why is here so many blank space @@

Reply:
Do you refer to this?:

    a       # => [0, 1, 2]

I've fixed it - if you refer to something else, let me know!